text/9999-event-vision.md

```diff
@@ -0,0 +1,60 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from traditional logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and traditional logs,
+and encourage the use of events over traditional logs.
```

Comment:
nit, but I'd make it even stronger and add something like

```suggestion
OpenTelemetry should have an Event API. This will help to promote the distinction between events and traditional logs. We will recommend all new instrumentation authors to use Event API instead of traditional logs.
```

I hope it can help resolve any future questions around "what's the right way to report events or logs"

Reply:
I added this below under "Interoperability with traditional logs"